common/src/logger/mod.rs

```diff
@@ -119,7 +119,7 @@ cfg_if::cfg_if! {
             fn basic_trace_time() {
                 let logger = create_test_logger("basic_trace_time".to_string());
 
-                slog_scope::scope(&logger.clone(), || {
+                slog_scope::scope(&logger, || {
```

Comment:
As requested by James, I reviewed what `slog_scope::scope` is doing here. I see why it looked sketchy, but I think it's actually OK. Because the `ScopeGuard` created in `slog_scope::scope` isn't returned, there's no way to keep the logger's pointer in `TL_SCOPES` past the execution of the closure, and you can't `drop(logger)` in the closure because the closure is called in the execution of `scope` while the borrowed `logger` argument is still in play, so you can't move the logger. So I don't think there's a way to make this Do Bad Things without doing something `unsafe` yourself (and at that point bets are mostly off anyway).